programs/ziti-edge-tunnel/include/model/events.h

```diff
@@ -85,8 +87,12 @@ XX(normal, __VA_ARGS__) \
 XX(connected, __VA_ARGS__) \
 XX(disconnected, __VA_ARGS__)
 
+#define INSTANCE_STATUS(XX, ...) \
+XX(ok, __VA_ARGS__)
```

Comment:
shouldn't it have more than one value?

Reply:
I am using this variable just to check whether the identity file is present in the config path. WDE maintains a config.json file which will have the fingerprint, name and other details of the identities. When we load that file, we need to use a variable that indicates which identity has the corresponding fingerprint file.